lmnet/executor/evaluate.py

```diff
@@ -140,6 +149,12 @@ def evaluate(config, restore_path):
     )
     validation_writer.add_summary(metrics_summary, last_step)
 
+    metrics_dict = {
+        'last_step': int(last_step),
+        'metrics': [{'name': k, 'value': float(v)} for k, v in zip(list(metrics_ops_dict.keys()), metrics_values)],
```

Comment:
Order of `list(dict.keys())` is not predictable. if `metrics_ops_dict` is a dict, I'm not sure if that will match with `metrics_values`.

Reply:
@tsawada 
Thanks!
I agree with you for readability but this issue is not only this PR, but we also have the same implementation on `lmnet/executor/train.py` and need to change another logics (lmnet/lmnet/utils.py).
I created the issue #704, better to solve another PR. Could you please allow let this is?


> that will match with metrics_values

In this implementation, only using `metrics_ops_dict.values()` and `metrics_ops_dict.keys()` and these orders is the same if there are no changes in `dict` objects. But it is not so good because this is implicit.

https://stackoverflow.com/questions/835092/python-dictionary-are-keys-and-values-always-the-same-order

Reply:
@tsawada 
Thanks!
I agree with you for readability but this issue is not only this PR, but we also have the same implementation on `lmnet/executor/train.py` and need to change another logics (lmnet/lmnet/utils.py).
I created the issue #704, better to solve another PR. Could you please allow let this is?


> that will match with metrics_values

In this implementation, only using `metrics_ops_dict.values()` and `metrics_ops_dict.keys()` and these orders is the same if there are no changes in `dict` objects. But it is not so good because this is implicit.

https://stackoverflow.com/questions/835092/python-dictionary-are-keys-and-values-always-the-same-order

Reply:
Can you add a TODO comment that this needs to be fixed?